CMakeLists.txt

```diff
@@ -228,7 +231,6 @@ if(CATKIN_ENABLE_TESTING)
   add_rostest(
     test/integration_tests/integrationtest_scan_topic.test DEPENDENCIES ${PROJECT_NAME}_integrationtest_scan_topic_node
   )
-
```

Comment:
Any reason for this newline to be deleted?

Reply:
No -> Reverted -> Done.